src/email.rs

```diff
@@ -93,7 +90,7 @@ pub fn request(app: &Config, email_addr: EmailAddress, client_id: &str, nonce: &
 pub fn verify(app: &Config, stored: &HashMap<String, String>, code: &str)
               -> BrokerResult<(String, String)> {
 
-    if code != &stored["code"] {
+    if code.to_lowercase() != (&stored["code"]).to_string() {
```

Comment:
We should probably apply the same transformation to both sides, otherwise we break all outstanding codes that have an upper-case letter when we deploy this.